dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/bnd.bnd

```diff
@@ -24,6 +24,12 @@ src: \
 
 fat.project: true
 
+# These features get added programatically
+tested.features: \
+  jaxrs-2.1,\
+  cdi-2.0,\
```

Comment:
if we are repeating these tests for JAX-RS 2.1 and CDI-2.0 also, lets take the feature version numbers out of the bucket name and just call it `com.ibm.ws.jaxrs.cdi_fat`